README.md

```diff
@@ -1,15 +1,16 @@
-# Java DropWizard Flyway Starter
+# The Extra Semicolon Backend Application Set Up
 
-Database Migration - Local
+Database Migration - Test
 ---
 
 1. Add your SQL script to `resources.db.migration` directory
 2. Add the following lines to your ~/.zshrc file:
 
 ```
-export FLYWAY_URL="jdbc:mysql://YOUR_DB_HOST/YOUR_DB_NAME"
-export FLYWAY_USER="YOUR_DB_USERNAME"
```

Comment:
You need both, DB_HOST for the Java application and FLYWAY_URL for your migrations

Reply:
There should be two sections, one that has DB_HOST, username and password and another with FLYWAY_URL, username, password and migrate